packages/fxa-auth-server/bin/key_server.js

```diff
@@ -163,6 +164,7 @@ function run(config) {
     )
     .then(() => {
       return {
+        server,
```

Comment:
Added for the test harness since there's no separate oauth-server wrapper anymore.